app/bundles/ProjectBundle/Translations/en_US/messages.ini

```
@@ -19,3 +19,6 @@ mautic.project.popover.updated.on="Updated on %date%"
 mautic.project.popover.view.details="View details"
 mautic.project.popover.edit="Edit"
 mautic.project.popover.footer.label="Project"
+mautic.project.contentblock.heading="Turn campaign plans into action"
+mautic.project.contentblock.subheading="One goal. One folder. All your related components."
```

```suggestion
mautic.project.contentblock.subheading="One goal. One folder. All your related resources."
```

As projects might include segments, components, campaigns, etc then I think we should not use the word 'components' here. I think resources might work better?

---

Good catch!